src/endpoint.rs

```diff
@@ -139,4 +148,43 @@ impl<S: MutEventSubscriber> RemoteEndpoint<S> {
         // own a `b: Box<T>` and call `*b`, the box goes away and we get the `T` inside.
         *result_box
     }
+
+    /// Call the specified closure on the associated local/remote `EventManager` (provided as a
+    /// `SubscriberOps` trait object), and discard the result. This method only fires
+    /// the request but do not wait for result, so it may be called from the same thread where
+    /// the event loop runs.
+    pub fn fire<F>(&self, f: F) -> Result<()>
+    where
+        F: FnOnce(&mut dyn SubscriberOps<Subscriber = S>) -> ErasedResult + Send + 'static,
```

Comment:
I think it looks cleaner if we remove the return value from the signature of `F` (since we don't have any way of interacting with it in this case). We can add it when building the `fnbox` below, so the inner signature continues to match.

Reply:
Good point:)